chef_master/source/server_backup_restore.rst

```diff
@@ -124,6 +127,76 @@ When restoring Chef server data, the previously backed-up files will be required
 
       $ chef-manage-ctl reconfigure
 
+
+Using Chef Backend
+=====================================================
+
+The backup and restore process allows you to restore a data backup
+into a newly built cluster in a disaster recovery scenario. It is not
+currently intended for the recovery of an individual machine in the
+chef-backend cluster or for point-in-time rollback of an existing
+cluster.
+
+Backup
+------
+
+To be able restore your data in the case of an emergency, you need
+backups of
+
+- the data in your Chef Backend cluster
+- the configuration from your Chef Server
```

Comment:
[nit] In line 6, it's "Chef server" (and IIRC docs.chef.io tends to use lowercase "server" consistently)